reference/components/payment_service_update_request.yaml

```
@@ -51,6 +51,7 @@ properties:
       two-letter code format.
     items:
       type: string
```

```suggestion
      type: string
      minLength: 2
      maxLength: 2
```

also needed for currencies above.